src/test/javascript/spec/component/organization/organization-management-resolve.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ActivatedRouteSnapshot } from '@angular/router';
+import { OrganizationManagementResolve } from 'app/admin/organization-management/organization-management-resolve.service';
+import { OrganizationManagementService } from 'app/admin/organization-management/organization-management.service';
+import { Organization } from 'app/entities/organization.model';
+import { of } from 'rxjs';
+
+describe('OrganizationManagementResolve', () => {
+    let organizationManagementService: OrganizationManagementService;
+    let organizationManagementResolve: OrganizationManagementResolve;
+
+    beforeEach(() => {
+        organizationManagementService = { getOrganizationById: jest.fn() } as any as OrganizationManagementService;
+        organizationManagementResolve = new OrganizationManagementResolve(organizationManagementService);
+    });
+
+    it('should search for organization by id', () => {
+        const toReturn = new Organization();
+        const spy = jest.spyOn(organizationManagementService, 'getOrganizationById').mockReturnValue(of(toReturn));
+
+        let result = undefined;
+        // @ts-ignore
+        organizationManagementResolve.resolve({ params: { id: 1 } } as any as ActivatedRouteSnapshot).subscribe((org) => (result = org));
+        expect(result).toBe(toReturn);
```

Comment:
```suggestion
        expect(result).toEqual(toReturn);
```
Since it is an object, it may be better to use `toEqual` instead of exact identity match.